numpy/core/src/umath/loops_unary_fp.dispatch.c.src

```diff
@@ -0,0 +1,294 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 sse41 avx2 avx512f
```

Comment:
```suggestion
 ** sse2 avx2 avx512f
```
No performance gain when SSE41 is enabled comparing to SSE2.

Reply:
Also, there's no performance gain for AVX2 and AVX512F for contiguous memroy access.

Reply:
therefore, I dropped three of them.